scripts/etkdg_ring_stress.py

```
@@ -0,0 +1,169 @@
+        base = {"mol_id": mid, "d": task["d"], "gap_kcal": task["gap_kcal"],
+                "n_flex_ring_atoms": len(ring_idx)}
+        if not ring_idx or crest_min is None:
+            base.update(note="no flexible ring / no reference min")
+            return [base]
+        calc = ref._gfnff_calc(task["method"], molh, charge)
+        tol = task["ring_rmsd_thr"]
+        for srt in (True, False):
+            confs = _embed(smiles, task["n_confs"], srt, task["seed"])
+            row = dict(base, use_small_ring_torsions=srt, n_embedded=len(confs))
+            if not confs:
+                row.update(note="embedding failed"); rows.append(row); continue
```

Ruff (0.15.21)

[error] Multiple statements on one line (semicolon)

(E702)